fast/stages/03-gke-multitenant/prod/main.tf

```diff
@@ -16,13 +16,26 @@
 
 locals {
   labels = merge(var.labels, { environment = var.environment })
-
-  _gke_robot_sa      = "serviceAccount:${module.gke-project-0.service_accounts.robots.container-engine}"
-  _cloud_services_sa = "serviceAccount:${module.gke-project-0.service_accounts.cloud_services}"
-  host_project_bindings = [
-    { role = "roles/container.hostServiceAgentUser", member = local._gke_robot_sa },
-    { role = "roles/compute.networkUser", member = local._gke_robot_sa },
-    { role = "roles/compute.networkUser", member = local._cloud_services_sa }
+  
+  _gke_robot_sas      = [
+    "serviceAccount:${module.gke-project-0.service_accounts.robots.container-engine}",
+    "serviceAccount:${module.gke-project-1.service_accounts.robots.container-engine}"
```

Comment:
Managing bindings for more than one project unnecessarily complex as this file is supposed to have a single project. The `gke-project-1` was a test I left behind accidentally. Please rebase your branch

Reply:
@juliocc I re-added the project-1, I think having a two projects scenario brings value for a multitenant setup, but, obviously, let's discuss about this.

Not sure about the things that I did with the binding, I did them cause weren't managed somewhere else, where the point where they are already managed?